src/Features/ERDDAP/Platform/Observations/WindCondition/index.tsx

```diff
@@ -1,90 +1,101 @@
+  platform: PlatformFeature
+}
+
+interface LoadingProps extends Props {
+  timeSeries: PlatformTimeSeries[]
+}
+
+interface DisplayProps extends LoadingProps {
   unit_system: UnitSystem
+  datasets: DataTimeSeries[]
 }
 
-export const windStandards = new Set(["wind_from_direction", "wind_speed", "wind_speed_of_gust"])
+/**
+ * Wind Observed conditions component
+ */
+export const ErddapWindObservedCondition: React.FunctionComponent<Props> = ({ platform }) => {
+  const { timeSeries } = pickWindTimeSeries(platform)
 
-export const ErddapWindObservedConditionBase: React.SFC<Props & SizeMeProps> = ({ platform, size, unit_system }) => {
-  // adjust number of barbs based on width
-  let barbsPerDay = 5
-  if (size && size.width && size.width < 800) {
-    barbsPerDay = 2
+  if (timeSeries.length < 0) {
+    return <WindError message="No wind data" />
   }
 
-  const windDatasets = platform.properties.readings.filter((reading) =>
-    windStandards.has(reading.data_type.standard_name)
-  )
+  return <LoadWindObservedConditionDisplay platform={platform} timeSeries={timeSeries} />
+}
 
-  const windTimeSeries: DataTimeSeries[] = windDatasets.map((reading) => ({
-    name: reading.data_type.long_name,
-    timeSeries: reading.readings,
-    unit: reading.data_type.units,
-  }))
-
-  Sentry.addBreadcrumb({
-    category: "ERDDAP Wind",
-    data: {
-      platformId: platform.id as string,
-      windDatasets,
-    },
-  })
-
-  if (windDatasets.length === 0) {
-    Sentry.captureMessage(("Unable to display wind data for " + platform.id) as string)
-    return noWind
+/**
+ * Load wind data
+ */
+export const LoadWindObservedConditionDisplay: React.FunctionComponent<LoadingProps> = ({ platform, timeSeries }) => {
+  const unit_system = useUnitSystem()
+  const { isLoading, data } = useDatasets(timeSeries)
+
+  if (isLoading) {
+    return (
+      <Row>
+        <Col>
+          <h5>Loading wind data in progress</h5>
+        </Col>
+      </Row>
+    )
   }
 
-  return (
-    <React.Fragment>
-      <ErddapDatasetStatus datasets={windDatasets} />
-      <ErddapDatasetLoader platformId={platform.id as string} datasets={windDatasets}>
-        <React.Fragment>
-          {windTimeSeries[0].timeSeries !== undefined && windTimeSeries[0].timeSeries.length > 0 ? (
-            <WindChart barbsPerDay={barbsPerDay} data={windTimeSeries} unit_system={unit_system} />
-          ) : null}
-        </React.Fragment>
-      </ErddapDatasetLoader>
-    </React.Fragment>
-  )
+  if (data) {
+    return <ErddapWindObservedConditionDisplay {...{ platform, unit_system, timeSeries }} datasets={data} />
+  }
+
+  return <WindError message="Error loading wind data" />
 }
 
-const noWind = (
-  <Row>
-    <Col>
-      <Alert color="warning">Unable to display wind info</Alert>
-    </Col>
-  </Row>
-)
+/**
+ * Display wind datasets
+ */
+export const ErddapWindObservedConditionDisplay: React.FunctionComponent<DisplayProps> = ({
+  platform,
+  unit_system,
```

Comment:
Codacy found an issue: [Identifier 'unit_system' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)